postcss.config.js

```diff
@@ -0,0 +1,11 @@
+const postcss = require('postcss');
```

Comment:
The thinking behind what I did is: Suit CSS is about the naming convention and importing maybe hence the `defaults`, and it can be extended with as many plugins as you want.

But I wouldn't be against abandoning this idea and move the config to a separate file,
in this case we'd still need to merge the configuration object with the user provided `root` and `browsers` or any other user provided `options`.

I'll let you decide on this :) I don't have any preference